src/frontend/utils/test/utdypendeVilkårsvurderinger.test.ts

```diff
@@ -0,0 +1,44 @@
+import { PersonType } from '../../typer/person';
+import { Resultat, UtdypendeVilkårsvurdering, VilkårType } from '../../typer/vilkår';
+import { UtdypendeVilkRsvurderingAvhengigheter } from '../utdypendeVilkårsvurderinger';
+import { erUtdypendeVilkårsvurderingerGyldig } from '../validators';
+
+const avhengigheter: UtdypendeVilkRsvurderingAvhengigheter = {
+    personType: PersonType.SØKER,
+    vilkårType: VilkårType.BOSATT_I_RIKET,
+    resultat: Resultat.OPPFYLT,
+    vurderesEtter: null,
+    brukErDeltBosted: true,
+    brukEøs: true,
+};
+
+describe('Utdypende Vilkårs', () => {
+    it('kan ikke velge UtdypendeVilkårsvurdering.DELT_BOSTED for VilkårType.BOSATT_I_RIKET', () => {
+        const actual = erUtdypendeVilkårsvurderingerGyldig(
+            [
+                UtdypendeVilkårsvurdering.VURDERING_ANNET_GRUNNLAG,
+                UtdypendeVilkårsvurdering.DELT_BOSTED,
+            ],
+            {
+                ...avhengigheter,
+                vilkårType: VilkårType.BOR_MED_SØKER,
+            }
+        );
+        expect(actual).toBe(true);
+    });
+
+    it('kan ikke velge UtdypendeVilkårsvurdering.DELT_BOSTED for VilkårType.BOSATT_I_RIKET', () => {
```

Comment:
"kan ikke velge UtdypendeVilkårsvurdering.DELT_BOSTED for VilkårType.BOSATT_I_RIKET"
Hva er forskjellen på denne og testen over?

Reply:
denne skal gi erGyldig false, siden det er forsøkt å legge på DELT_BOSTED for et BOSATT_I_RIKET vilkår